code/game/machinery/rechargestation.dm

```diff
@@ -97,13 +97,19 @@
 	update_icon()
 
 /obj/machinery/recharge_station/proc/process_occupant()
-	if(occupant && iscyborg(occupant))
+	if(!occupant)
+		return
+	if(iscyborg(occupant))
 		var/mob/living/silicon/robot/R = occupant
 		restock_modules()
 		if(repairs)
 			R.heal_bodypart_damage(repairs, repairs - 1)
 		if(R.cell)
 			R.cell.charge = min(R.cell.charge + recharge_speed, R.cell.maxcharge)
+	if(isethereal(occupant))
+		var/mob/living/carbon/human/H = occupant
+		var/datum/species/ethereal/E = H.dna?.species

```

Comment:
What's the point of null conditional if you're going to runtime on next line anyway.

Reply:
because im retarded